test/LibraryService.test.ts

```diff
@@ -0,0 +1,46 @@
+import { initialize, Server } from '@projecttacoma/node-fhir-server-core';
+import { serverConfig } from '../src/config/serverConfig';
+import { cleanUpDb, testSetup } from './utils';
+const supertest = require('supertest');
+
+let server: Server;
+
+const Library: fhir4.Library = {
+  resourceType: 'Library',
+  type: { coding: [{ code: 'logic-library' }] },
+  id: 'library123',
+  status: 'active'
+};
+
+describe('LibraryService', () => {
```

Comment:
Seems like these were mostly copied from deqm-test-server, which is fine, but can we get a fresh start on some of the formatting? Specifically the `it` over `test` and test naming conventions here: https://gist.github.com/mgramigna/bdc933fae2150216681c28d1f0f7aac7#testing

Reply:
Oh yes how quickly I forgot about that...good thing we have the doc